modules/spacelift/README.md

```diff
@@ -97,20 +97,23 @@ chmod +x rootfs/usr/local/bin/spacelift*
 **NOTE**: this won't be necessary for Kubernetes implementations once Spacelift adds support for Kubernetes worker pools.
 
 1. Clone the following GitHub repository: `git clone git@github.com:spacelift-io/spacelift-worker-image.git`
-2. In the cloned repo, apply the [contrib/spacelift.pkr.hcl.patch](contrib/spacelift.pkr.hcl.patch) patch
-
-   ```bash
-   git apply spacelift.pkr.hcl.patch
-   ```
-
-   Change the region as needed, but make sure you update the `base_ami` with the latest Amazon Linux 2 AMI id:
-
-   ```bash
-   aws ec2 describe-images --owners amazon --filters "Name=name,Values=amzn2-ami-hvm-2.0.202*-x86_64-gp2" --query 'sort_by(Images, &CreationDate)[-1]'
-   ```
-
-3. Run `packer build spacelift.pkr.hcl` against the account Spacelift will be deployed to
-4. Take note of the AMI created and use it to configure the `spacelift-worker-pool` component
+1. Grab the latest AL2 AMI ID:
+```bash
+aws ec2 describe-images --region $YOUR_TARGET_REGION --owners amazon --filters "Name=name,Values=amzn2-ami-hvm-2.0.202*-x86_64-gp2" --query 'sort_by(Images, &CreationDate)[-1]' | jq -r .ImageId
+```
+1. In the cloned repo, add a `default.hcl` file with the following content:
+```hcl
+base_ami = "$IMAGE_ID_FROM_ABOVE
```

Comment:
Please add explanation about these variables. It is not clear whether you mean to say that these variables should be set somewhere (in the environment, in a config file, etc.) or are placeholders you should replace with literal values. I am in favor of using `<placeholder>` rather than `$PLACEHOLDER` if the intent is to make a literal replacement. You can also use a real value and note both in the narrative text above and in the code via a comment that it should be replaced, e.g.

Replace `us-east-1` with the AWS region where you intend to deploy the worker pool:
```hcl 
region = "us-east-1" # replace with your target region
```

In any case, you left out a trailing quotation mark:
```suggestion
base_ami = "$IMAGE_ID_FROM_ABOVE"
```

Reply:
@Nuru good feedback -- updated 👍